src/daq/Notification.cpp

```diff
@@ -42,31 +44,70 @@ void Notification::start() {
     LOG_INF << "Notification stream started";
 }
 
-void Notification::block(Info::MODE mode,
-                         const std::string image_id,
-                         const std::string folder) {
+void Notification::block(Info::MODE mode, const std::string image_id, const std::string folder) {
 
     if (mode != Info::MODE::LIVE) {
         return;
     }
 
-    LOG_DBG << "IMS::Image blocking for image " << image_id;
-    IMS::Image image(*_store, *_stream, TIMEOUT);
-    LOG_DBG << "Acquired image " << image_id;
+    // We make the assumption that we open the stream on start up of the Forwarder.  This
+    // is before getting an endReadout (which calls this method), so we can't be 
+    // "ahead" of reading the data coming from the stream, only behind. Any images that
+    // were on the stream that we skip will be handled by the catchup archiver.
+    //
+    while (1) {
```

Comment:
According to LSST Dev Guide `while (true)` form should be used for inifinte loops (see [5-18](https://developer.lsst.io/cpp/style.html?highlight=null#the-form-while-true-should-be-used-for-infinite-loops)).